frontend/src/components/2DModel/Modelization.vue

```diff
@@ -46,6 +46,12 @@ import {
 } from "src/components/Monaco/svg_maths";
 import EventBus from "src/services/EventBus";
 
+function randomString({ length }) {
+	return Array.from({ length }, () =>
+		Math.floor(Math.random() * 16).toString(16)
```

Comment:
Please fix the security hotspots associated to this line

https://sonarcloud.io/project/security_hotspots?id=ditrit_leto&pullRequest=200&resolved=false&types=SECURITY_HOTSPOT